GameData/ROCapsules/PartConfigs/MercuryBDB/MercuryLES.cfg

```diff
@@ -0,0 +1,156 @@
+PART
+{
+	name = ROC-MercuryLESBDB
+	module = Part
+	author = CobaltWolf, capkirk
+
+	RSSROConfig = true
+
+	MODEL
+	{
+		model = ROCapsules/Assets/BDB/Mercury/bluedog_Mercury_LES
+		rotation = 0, 180, 0
+	}
+	rescaleFactor = 1.512
+	node_stack_bottom = 0.0, -0.6293343, 0.0, 0.0, -1.0, 0.0, 0
+	CoMOffset = 0, 1.4, 0
+	//CoPOffset = 0, -0.63, 0
+	sound_vent_large = decouple
+	fx_gasBurst_white = 0.0, -0.269, 0.0, 0.0, -1.0, 0.0, decouple
+	TechRequired =	engineering101
+	entryCost = 4500
+	cost = 350
+	category = Utility
+	subcategory = 0
+	title = Mercury Launch Escape System
+	manufacturer = Grand Central Rocket Company
+	description = This solid rocket tower is designed to wrench the command pod up and away from a malfunctioning rocket. Use only in case of emergency. WARNING: Staging the LES will also decouple it from the bottom node, in order to dispose of it after launch. Map the engine to the ABORT action group for emergency use.
+	attachRules = 1,0,1,1,0
+	mass = 0.4679
+	// heatConductivity = 0.04 // 1/3 default
+	dragModelType = default
+	maximum_drag = 0.3
+	minimum_drag = 0.2
+	angularDrag = 2
+	crashTolerance = 15
+	breakingTorque = 30
+	breakingForce = 30
+	maxTemp = 673.15
+	skinMaxTemp = 773.15
```

Comment:
@lpgagnon You should test this with the FAR + my heating updates.  Those might be low skin temps for something that will need to survive some convection during launch.  I'd definitely suggest leaving the heatConductivity = 0.04 commented out.